<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="gvMain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>127</red>
       <green>127</green>
       <blue>127</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>540</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>触发模式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>曝光设置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>540</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>抓拍设置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>540</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>连续采集并保存ROI</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>211</width>
      <height>60</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>增益</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>曝光时间（微秒）</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_gain">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>8</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_exposure_time">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>33</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="singleStep">
      <double>10.000000000000000</double>
     </property>
     <property name="stepType">
      <enum>QAbstractSpinBox::DefaultStepType</enum>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>560</y>
      <width>131</width>
      <height>90</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButton_collect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>连续采集</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_software_trigger">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>软触发</string>
     </property>
    </widget>
    <widget class="QPushButton" name="software_trigger_once_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>100</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>软件触发一次</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_14">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>560</y>
      <width>250</width>
      <height>90</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_snap_catch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>捕获</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>路径：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="snap_path_lineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>15</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_snap_path">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>15</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_15">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>560</y>
      <width>261</width>
      <height>90</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_captureROI_start">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>50</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>路径：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="captureROI_path_lineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>15</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_captureROI_path">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>15</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_captureROI_stop">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>50</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_captureROI_num">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>55</y>
       <width>70</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>采集数量</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="gvMain_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>127</red>
       <green>127</green>
       <blue>127</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_camera_select">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1052</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
